toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/GeoViewProxy.kt

```diff
@@ -54,6 +54,14 @@ public sealed class GeoViewProxy(className: String) {
     private val nullGeoViewErrorMessage: String =
         "$className must be part of the composition when this member is called."
 
+    /**
+     * True if continuous panning across the international date line is enabled in the GeoView, false otherwise.
+     *
+     * @since 200.4.0
+     */
+    public val isWrapAroundEnabled: Boolean?
+        get() = geoView?.isWrapAroundEnabled
```

Comment:
The return value should not be nullable. Since properties should not throw, it makes sense to return `false` in the case of `geoView` being null: 
```suggestion
    public val isWrapAroundEnabled: Boolean
        get() = geoView?.isWrapAroundEnabled ?: false
```